src/server.rs

```diff
@@ -765,9 +772,9 @@ impl Builder {
     /// # {
     /// // `server_fut` is a future representing the completion of the HTTP/2.0
     /// // handshake.
-    /// let handshake_fut = Builder::new()
+    /// let server_fut = Builder::new()
```

Comment:
i can't comment on the top of this block but the intro maybe better as _Configures the send-payload data type._